cengage/react-magma
# Fix/simple pagination

Issue: # [991](https://github.com/cengage/react-magma/issues/991)

## What I did
- Fixed spacing between buttons.
- Fixed VoiceOver to properly read Page X of X selected.
- Fixed Tooltip to display inverse colors when isInverse is enabled on Pagination.

## Screenshots (if appropriate):
![stuff](https://user-images.githubusercontent.com/77400920/232780733-84589f71-1b4c-49fc-b211-0c1d8fbacde9.png)

## Checklist 
- [x] NA
- [x] NA

## How to test

- Confirm proper spacing of 16 pixels between navigations buttons and surrounding elements on either side.
- Confirm Tooltip displays appropriate inverse color when isInverse is enabled on Pagination
- Confirm VoiceOver properly reads Page X of X selected when keyboard navigation tabs through the select and then also when a selection is made.
